File: src/PCB.py
class PCB:
	def __init__(self, id, et, at):
		self.pid = id
		self.pST = et
		self.pAT = at


with open('Simulation1-data.txt','r') as data:
	lines = [line.split() for line in data]


lines = [[int(e) for e in line] for line in lines]



processes = []
for line in lines:
	pcb = PCB(line[0],line[1],line[2])
	processes.append(pcb)

File: src/PCB.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
/**
 *
 * @author jarro_000
 */
public class PCB 
{
    private int pid;
    private int pST;
    private int pAT;
    
    public PCB(int id, int eT, int aT)
    {
        pid = id;
        pST = eT;
        pAT = aT;
    }
    
    public int getID()
    {
        return this.pid;
    }
    
    public int getST()
    {
        return this.pST;
    }
    
    public int getAT()
    {
        return this.pAT;
    }
    
}
